values.yaml

```diff
@@ -88,9 +88,14 @@ deployment:
   securityContext:
     # -- Run as user ID
     runAsUser: 802
```

Comment:
Need to get 802 to a new key under runAsUser and update the reference in the toml